.github/workflows/pr.yaml

```diff
@@ -318,7 +318,7 @@ jobs:
   unit-tests-cudf-pandas:
     needs: [wheel-build-cudf, changed-files]
     secrets: inherit
-    uses: rapidsai/shared-workflows/.github/workflows/wheels-test.yaml@branch-25.06
+    uses: rapidsai/shared-workflows/.github/workflows/wheels-test.yaml@python-3.13
```

Comment:
We have merged `python-3.13` into `branch-25.06`. Please revert this to `branch-25.06`. https://github.com/rapidsai/shared-workflows/pull/268

Reply:
Thanks @bdice !